README.md

```diff
@@ -111,21 +127,19 @@ $ geth --help
           blocks. For example, if a slot is 12 seconds long, and the offset is 2 seconds,
           the builder will submit blocks at 10 seconds into the slot.
           [$FLASHBOTS_BUILDER_SUBMISSION_OFFSET]
-
-    --builder.validation_blacklist value
```

Comment:
for backwards compatibility, should also mark this as DEPRECATED before removing

Reply:
ah ok that's fair

Reply:
updated